packaging/osx/buildpackage.sh

```diff
@@ -14,7 +14,7 @@ if [ -e "OpenRA.app" ]; then
     exit 2
 fi
 
-wget https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${LAUNCHER_TAG}/launcher.zip || exit 3
+curl -s -L -o launcher.zip https://github.com/OpenRA/OpenRALauncherOSX/releases/download/${LAUNCHER_TAG}/launcher.zip || exit 3
```

Comment:
Why didn't you come up with that last week. Now this is buried by lot's of other commits.